data/mods/Aftershock/items/ammo/6.5mm.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "id": "afs_6.5mm_caseless",
+    "type": "AMMO",
+    "name": { "str_sp": "6.5mm caseless" },
+    "description": "An intermediate bullet, popularized during the first years of the Reclamation, these bullets are interchangeable between most modern assault rifles and machine guns.  These rounds being caseless cannot be disassembled or reloaded.",
+    "weight": "12 g",
+    "volume": "194 ml",
+    "price": 280,
+    "price_postapoc": 900,
+    "material": [ "brass", "powder" ],
+    "symbol": "=",
+    "color": "yellow",
+    "count": 30,
+    "stack_size": 47,
+    "ammo_type": "afs_6.5mm",
+    "range": 36,
+    "damage": { "damage_type": "bullet", "amount": 44, "armor_penetration": 2 },
+    "dispersion": 30,
+    "recoil": 1500,
+    "effects": [ "COOKOFF" ]
+  },
+  {
+    "id": "afs_6.5mm_lr",
+    "type": "AMMO",
+    "name": { "str_sp": "6.5mm LR" },
+    "description": "6.5mm Long Rifle ammunition with unjacketed bullets.  The 6.5mm LR round is extremely weak with very low stopping power, short range, and negligible recoil.  It is most extremely cheap to manufacture, and is only useful for rifle training, and hunting small animals.",
```

Comment:
These aren't caseless?

They probably should be as the other 6.5 guns can fire them alright.

Reply:
Ah yes, they are caseless

Reply:
I 'm not sure about that switch. It seems likely that the design considerations to make an hybrid gun like that would have serious to severe drawbacks. So it likely wouldnt happen even with very good tech.

Mostly I say this because I'm reasonably sure that players that do like guns are going to complain about this.

Reply:
I suspect that it would be possible to convert the weapons, but it wouldn't be as simple as flipping a switch - replacing the loading mechanism at minimum.